src/lqcut.jl

```diff
@@ -159,11 +159,29 @@ function lq_cut(
     dep_µ::Unitful.Energy, dep_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=3.5, uncertainty::Bool=true
     )
 
-    # define sidebands
-    lq_dep = lq_classifier[dep_µ - dep_sideband_sigma * dep_σ .< e_cal .< dep_µ + dep_sideband_sigma * dep_σ]
-    lq_sb1 = lq_classifier[dep_µ -  2 * dep_sideband_sigma * dep_σ .< e_cal .< dep_µ - dep_sideband_sigma * dep_σ]
-    lq_sb2 = lq_classifier[dep_µ + dep_sideband_sigma * dep_σ .< e_cal .< dep_µ + 2 * dep_sideband_sigma * dep_σ]
-    
+    # define sidebands; different for low and high energy resolution detectors to avoid sb reaching into 212-Bi FEP
+    DEP_edge_left  = dep_µ - dep_sideband_sigma * dep_σ
+    DEP_edge_right = dep_µ + dep_sideband_sigma * dep_σ
+
+    if dep_σ < 2.0u"keV"
+        sb1_edge = dep_µ - 2 * dep_sideband_sigma * dep_σ
+        sb2_edge = dep_µ + 2 * dep_sideband_sigma * dep_σ  
+
+        lq_dep = lq_classifier[DEP_edge_left .< e_cal .< DEP_edge_right]
+        lq_sb1 = lq_classifier[sb1_edge .< e_cal .< DEP_edge_left]
+        lq_sb2 = lq_classifier[DEP_edge_right .< e_cal .< sb2_edge]
+    else
+        sb1_edge = dep_µ - 2 * dep_sideband_sigma * dep_σ  
+        sb2_edge = dep_µ - 3 * dep_sideband_sigma * dep_σ
+
+        lq_dep = lq_classifier[DEP_edge_left .< e_cal .< DEP_edge_right]
+        lq_sb1 = lq_classifier[sb1_edge .< e_cal .< DEP_edge_left]
+        lq_sb2 = lq_classifier[sb2_edge .< e_cal .< sb1_edge]
+    end
```

Comment:
Are there cases where `σ` ends up being so high, that it reaches into the peak to the left?